scrapper.py

```diff
@@ -44,20 +78,106 @@ def get_search_urls(self):
         pass
 
 
+class HTMLParser:
+    def __init__(self, article_url, article_id):
+        self.article_url = article_url
+        self.article_id = article_id
+        self.article = Article(self.article_url, self.article_id)
+
+    def _fill_article_with_meta_information(self, article_bs):
+        # DATE
+        # the second date - of publication
+        datetime_of_publication = article_bs.find('span', class_='echo_date')['data-published']
+        self.article.date = datetime.strptime(datetime_of_publication, '%Y-%m-%dT%H:%M:%S%z')
+
+        # AUTHOR
+        try:
+            self.article.author = article_bs.find('div', class_='meta-item meta-item_author').text.strip()
+        except AttributeError:
+            self.article.author = 'NOT FOUND'
+        # TOPICS
+        self.article.topics = 'NOT FOUND'
```

Comment:
According to docs, in case there are not any topics or keywords present in your source, list is supposed to be left empty.